src/Cardano/Shell/Constants/CLI.hs

```diff
@@ -194,3 +198,76 @@ configBlockCLIParser =
           <> metavar "FIXED-TIME-CQ"
           <> help "Chain quality will be also calculated for this amount of seconds."
            )
+
+--------------------------------------------------------------------------------
+-- Certificates
+--------------------------------------------------------------------------------
+
+-- | TLS CLI Parser.
+configTLSCLIParser :: Parser PartialTLS
+configTLSCLIParser =
+    PartialTLS
+        <$> lastOption configCertificateCLIParser
+        <*> lastOption configCertificateCLIParser
+        <*> lastOption configCertificateCLIParser
+
+-- | Certificate CLI parser.
+configCertificateCLIParser :: Parser PartialCertificate
+configCertificateCLIParser =
+    PartialCertificate
+        <$> option auto
```

Comment:
We're not allowing defaulting here (`lastOption`)?

Reply:
Yep, turns out I need to fix a couple of things...

Reply:
Turns out those "couple of things" are an issue.
Need to rewrite the config and we need to keep the configuration above two levels deep if we want to use this "mechanism".